### PR TITLE
ctlv3: add 'print-value-only' flag to get command

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/6234.

/cc @xiang90 @heyitsanthony

### CODE REVIEW

File: etcdctl/ctlv3/ctl.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379"}, "gRPC endpoints")
 
-	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (json, proto, simple, table)")
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple-value, json, proto, simple, table)")
```
Comment (xiang90) [inline etcdctl/ctlv3/ctl.go]: i feel we probably should add a new flag to kv operations. simple-value does not make sense for other operations like membership management.

Comment (heyitsanthony) [inline etcdctl/ctlv3/ctl.go]: alphabetize list

Comment (gyuho) [inline etcdctl/ctlv3/ctl.go]: @xiang90 How about only adding `cmd.Flags().BoolVar(&printValueOnly, "print-value-only", false, "Print value only")` to get command?

Comment (heyitsanthony) [inline etcdctl/ctlv3/ctl.go]: @xiang90 a separate option for kv options doesn't make sense for other printers though; what would be given for something like `-value-only -w protobuf`?

**Comment by gyuho** [comment]
@heyitsanthony @xiang90 PTAL. Thanks.

File: etcdctl/README.md [hunk 2]
```diff
@@ -69,6 +69,8 @@ GET gets the key or a range of keys [key, range_end) if `range-end` is given.
 
 - rev -- specify the kv revision
 
+- print-value-only -- print only value when used with write-out=simple
```
Comment (xiang90) [inline etcdctl/README.md]: probably just value-only

**Comment by xiang90** [comment]
lgtm. defer to @heyitsanthony

File: etcdctl/README.md (hunk 2 above)
Comment (heyitsanthony) [inline etcdctl/README.md]: `--write-value-only`? so it's consistent with `-w` / `--write-out`

Comment (xiang90) [inline etcdctl/README.md]: fine with me.

File: etcdctl/ctlv3/command/get_command.go [hunk 3]
```diff
@@ -49,6 +50,7 @@ func NewGetCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&getFromKey, "from-key", false, "Get keys that are greater than or equal to the given key")
 	cmd.Flags().Int64Var(&getRev, "rev", 0, "Specify the kv revision")
 	cmd.Flags().BoolVar(&getKeysOnly, "keys-only", false, "Get only the keys")
+	cmd.Flags().BoolVar(&printValueOnly, "print-value-only", false, "Print only value when used with write-out=simple")
```
Comment (heyitsanthony) [inline etcdctl/ctlv3/command/get_command.go]: `Only write values when using the "simple" output format`?

**Comment by heyitsanthony** [comment]
lgtm after fixing the minor nits

**Comment by gyuho** [comment]
Fixed. Merging in greens. Thanks!

**Pull request merged**